simbad/lattice/latticesearch.py

```diff
@@ -283,12 +284,18 @@ def copy_results(self, source, destination):
 
         to_del = []
         for count, result in enumerate(self.results):
+            f_tmp_out = tempfile.NamedTemporaryFile(suffix='.pdb', delete=False)
             f_name = os.path.join(source, '{0}', 'pdb{1}.ent.gz').format(result.pdb_code[1:3].lower(),
                                                                          result.pdb_code.lower())
             f_name_out = os.path.join(destination, '{0}.pdb'.format(result.pdb_code))
             try:
-                with gzip.open(f_name, 'rb') as f_in, open(f_name_out, 'w') as f_out:
-                    f_out.write(f_in.read())
+                with gzip.open(f_name, 'rb') as f_in:
+                    f_tmp_out.write(f_in.read())
+                PS = simbad.util.pdb_util.PdbStructure(f_tmp_out.name)
```

Comment:
Why the capital variable name? I'd prefer something like ``struct`` or something ...

Reply:
I meant to change this after testing that it worked and forgot!